apps/wallet/src/utils/helper.utils.ts

```diff
@@ -532,3 +532,13 @@ export const transformData = (
 
   return normalizedInput;
 };
+
+export function hexStringToUin8Array(input: string) {
```

Comment:
```suggestion
export function hexStringToUint8Array(input: string) {
```

Reply:
Thanks, fixed this in a separate commit since the references to this function needed to be modified as well.